pkg/size/size.go

```diff
@@ -0,0 +1,460 @@
+package size
+
+import (
+	sizev1 "github.com/blackducksoftware/synopsys-operator/pkg/api/size/v1"
+	types2 "github.com/blackducksoftware/synopsys-operator/pkg/apps/types"
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"strings"
+)
+
+// GetSize returns the size for each container
+func GetSize(name string) *sizev1.Size {
```

Comment:
Suggest making this function name and comment more suited to what this is actually doing.  These are now the default size definitions.